src/proto/FunctionRpc.proto

```diff
@@ -306,6 +306,8 @@ message BindingInfo {
 
   // Direction of the given binding
   Direction direction = 3;
+
+  string dataType = 4;
```

Comment:
change this to a enum - Matching [DataType](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Description/Binding/DataType.cs) in functions host